configure.ac

```diff
@@ -96,6 +96,8 @@ fi
 CPPFLAGS="$CPPFLAGS $XML2HEAD"
 
 PKG_CHECK_MODULES(GLIB, [glib-2.0])
+PKG_CHECK_MODULES(PCMK, [pacemaker],,
+		  [PKG_CHECK_MODULES(PCMK, [pcmk])])
```

Comment:
Whenever new autotools function is used, it'd make sense to use the
proper `[quoting]` everywhere, regardless of the pre-existing weird mix.

Also, since booth is to deal (AFAIK) only with this `pacemaker-service`
part, I think this shall in fact be:

```
PKG_CHECK_MODULES([PCMK], [pacemaker-service])
```

Also, you can notice that Travis CI (as a sort of a paradox here) got
broken with this change.

The way forward with the above proposed change would be to
`s/libcrmcluster4-dev/libcrmservice-dev/` in `.travis.yml`.